ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -7,6 +7,7 @@ declaration template components/ccm/schema;
 
 include 'quattor/types/component';
 include 'pan/types';
+include 'components/accounts/functions';
```

Comment:
it can be a kept group too (like one created by an rpm etc). i don't think it's overly protective given the central role of ccm.
however, you will enter chicken-and-egg world if the group can't be found and quattor is supposed to config ldap or nss.
the current ccm code fails hard the group can't found. maybe it should fall back to root-only (so the system is not unusable?)